server_admin/topics/clients/proc-creating-oidc-client.adoc

```diff
@@ -0,0 +1,116 @@
+[id="proc-creating-oidc-client_{context}"]
+==== Creating an Open ID Connect Client
+
+. Click the `Clients` left menu item.  
+. Click the `Create` button. This will bring you to the `Add Client` page.
+
+.Clients
+image:{project_images}/clients.png[Clients]
+
+.Add Client
+image:{project_images}/add-client-oidc.png[Add Client]
+
+Enter in the `Client ID` of the client.  This should be a simple
+alpha-numeric string that will be used in requests and in the {project_name} database to identify the client.
+Next select `openid-connect` in the `Client Protocol` drop down box.
+Finally enter in the base URL of your
+application in the `Root URL` field and click `Save`.  This will create the client and bring you to the client `Settings`
+tab.
+
+.Client Settings
+image:{project_images}/client-settings-oidc.png[Client Settings]
+
+Let's walk through each configuration item on this page.
+
+*Client ID*
+
+The alpha-numeric string that will be used as the client identifier for OIDC requests.
+
+*Name*
+
+The display name for the client whenever it is displayed in a {project_name} UI screen.  You can localize
+the value of this field by setting up a replacement string value i.e. $\{myapp}.  See the link:{developerguide_link}[{developerguide_name}]
+for more information.
+
+*Description*
+
+The description of the client.  This can also be localized.
+
+*Enabled*
+
+When turned off, the client will not be allowed to request authentication.
+
+*Consent Required*
+
+If this is on, then users will get a consent page which asks the user if they grant access to that application.  It will also display the metadata that the client is interested in so that the user knows exactly what information the client is getting access to.
```

Comment:
Use consistent phrasing. "When turned on users see a consent page that they can use to grant ...."
"What is interested"? How can the client be interested in anything? Do you mean the client uses?